user_modules/IsReady_Template/isReady.lua

```diff
@@ -272,7 +272,7 @@ end
 	fileContent  =  string.gsub(fileContent, '".?default.?".?:.?.?%{.-%}', DefaultContant)
 
 
-	f = assert(io.open(config.pathToSDL.. "/sdl_preloaded_pt.json", "w+"))
+	f = assert(io.open(commonPreconditions:GetPathToSDL().. "/sdl_preloaded_pt.json", "w+"))
```

Comment:
@istoimenova, Please remove "/" from "/sdl_preloaded_pt.json" since it will be added by GetPathToSDL()